### PR TITLE
Ref task #6981 Index category

### PR DESCRIPTION
**Pull request by leducanh120197**
## Redmine URL
https://edu-redmine.framgia.vn/issues/6981

## Other
pull13_rubocop
pull14

### CODE REVIEW

**Comment by leducanh120197** [comment]
ready

**Comment by leducanh120197** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -3,4 +3,8 @@ def full_title page_title = ""
     base_title = t "helpers.application.title"
     page_title.empty? ? base_title : page_title + " | " + base_title
   end
+
+  def categories_list
+    Category.select(:id, :name)
```
Comment (thanhhungchu95) [inline app/helpers/application_helper.rb]:
```suggestion
    Category.select :id, :name
```

**Review by thanhhungchu95 — COMMENTED** [review]
last comment

**Comment by leducanh120197** [comment]
ready

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -3,4 +3,8 @@ def full_title page_title = ""
     base_title = t "helpers.application.title"
     page_title.empty? ? base_title : page_title + " | " + base_title
   end
+
+  def categories_list
+    Category.select :id, :name
```
Comment (yanghoxom) [inline app/helpers/application_helper.rb]: `pluck` thôi không cần dùng select

**Comment by leducanh120197** [comment]
ready

**Pull request merged**